vulnerabilities/importers/__init__.py

```diff
@@ -27,6 +28,7 @@
     pysec.PyPIImporter,
     debian.DebianImporter,
     gitlab.GitLabAPIImporter,
+    fireeye.FireyeImporter,
```

Comment:
Since we don't know license for this, Make a new list called `IMPORTERS_WITH_UNKNOWN_LICENSE` and add Fireeye in that list

Reply:
We now know the license:  See https://github.com/mandiant/Vulnerability-Disclosures/issues/5#issuecomment-1279553128

> I have updated our readme and source files (*.cpp). CVE information is CC BY-SA 4.0 and source files are MIT. Please let me know if this resolves this issue to satisfaction. Thanks! --Aaron

And https://github.com/mandiant/Vulnerability-Disclosures#mandiant-vulnerability-disclosures